lib/parliament/utils/config/initializers/sanitizer.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.after_initialize do
+  ActionView::Base.sanitized_allowed_tags += %w(table thead tbody tfoot tr th td iframe)
+  ActionView::Base.sanitized_allowed_attributes += %w(allowfullscreen seamless frameborder style)
```

Comment:
Same applies for the style attribute - I'd prefer it if you could call Sanitize with overrides if possible?